src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -519,17 +520,21 @@ public EmailWrapper generateStudentCourseRejoinEmailAfterGoogleIdReset(
     }
     
     /**
-     * Generates the course join email for the given {@code instructor} in {@code course}.
+     * Generates the course join email for the given {@code inviteReceiver} for {@code course}.
+     * Also specifies contact information of {@code inviter} instructor or admin.
      */
-    public EmailWrapper generateInstructorCourseJoinEmail(CourseAttributes course, InstructorAttributes instructor) {
+    public EmailWrapper generateInstructorCourseJoinEmail(AccountAttributes inviter,
+            InstructorAttributes inviteReceiver, CourseAttributes course) {
```

Comment:
Keep the variable name as `instructor`. The method signature and method name would already suggest that `instructor` is the one being invited.